api/src/main/java/org/pmiops/workbench/api/UserController.java

```diff
@@ -60,11 +62,13 @@ public UserController(
       Provider<WorkbenchConfig> configProvider,
       FireCloudService fireCloudService,
       UserService userService,
-      @Qualifier(END_USER_CLOUD_BILLING) Provider<Cloudbilling> cloudBillingProvider) {
+      @Qualifier(END_USER_CLOUD_BILLING) Provider<Cloudbilling> cloudBillingProvider,
```

Comment:
I would really like to move listBillingAccounts() and these 2 dependencies to a new BillingAccountService, but I found that doing so obscured the intent of these changes.  I'll probably file a followup PR for that.